data/fwupd.conf.md

```diff
@@ -0,0 +1,238 @@
+    If set to "*", all domains are verbose, which is the same as running the daemon with **--verbose --verbose**.
+
+**UpdateMotd=**[BOOL]
+
+    Update the message of the day (MOTD) on device and metadata changes.
+
+**EnumerateAllDevices=**[BOOL]
+
+    For some plugins, enumerate only devices supported by metadata.
+
+**ApprovedFirmware=**[HASH];[HASH];[HASH]
+
+    A list of firmware checksums that has been approved by the site admin
+    If unset, all firmware is approved.
+
+**BlockedFirmware=**[HASH];[HASH];[HASH]
+
+    Allow blocking specific devices by their cabinet checksum, either SHA-1 or SHA-256.
+
+**UriSchemes=**
+
+    Allowed URI schemes in the preference order; failed downloads from the first scheme will be retried with the next in order until no choices remain.
+    If unset or no schemes are listed, the default will be: **file,https,http,ipfs**.
+
+**IgnorePower=**[BOOL]
+
+    Ignore power levels of devices when running updates.
+
+**OnlyTrusted=**[BOOL]
+
+    Only support installing firmware signed with a trusted key.
+    Do not enable this on a production or trusted system.
+
+**ShowDevicePrivate=**[BOOL]
+
+    Show private data such as device serial numbers and instance IDs to clients.
+
+**AllowEmulation=**[BOOL]
+
+    Allow capturing and loading device emulation by logging all USB transfers.
+    Enabling this will greatly increase the amount of memory fwupd uses when upgrading devices.
+
+**TrustedUids=**[INTEGER_UIDS]
+
+    UIDs matching these values that call the D-Bus interface should marked as trusted.
+
+**HostBkc=**[STRING]
+
+    A host best known configuration is used when using `fwupdmgr sync` which can downgrade firmware to factory versions or upgrade firmware to a supported config level. e.g. **vendor-factory-2021q1**
+
+**EspLocation=**[PATH]
+
+    Override the location used for the EFI system partition (ESP) path.
+    This is typically used if UDisks is not available, or was not able to automatically identify the location for any reason.
+
+**Manufacturer=**
+
+**ProductName=**
+
+**ProductSku=**
+
+**Family=**
+
+**EnclosureKind=**
+
+**BaseboardProduct=**
+
+**BaseboardManufacturer=**
+
+    Override values for SMBIOS or Device Tree data on the local system.
+    These are only required when the SMBIOS or Device Tree data is invalid, missing, or to simulate running on another system.
+    Empty values should be used to populate blank entries or add values to populate specific entries.
+
+UEFI_CAPSULE PARAMETERS
+=======================
+
+The `[uefi_capsule]` section can contain the following parameters.
+
+**EnableGrubChainLoad=**[BOOL]
+
+    Use GRUB to launch `fwupdx64.efi`.
+
+**DisableShimForSecureBoot=**[BOOL]
+
+    The shim loader is required to chainload the fwupd EFI binary unless the `fwupd.efi` file has been self-signed manually.
+
+**RequireESPFreeSpace=**[SIZE_IN_MB]
+
+    Amount of free space required on the ESP, for example using `32` for 32Mb.
```

Comment:
I think the actual default is useful to put here, no?

Reply:
I'm trying to avoid it -- I want the default to be specified in one place -- in the code. If the distro changes the default in the code for whatever reason I don't want the two to be "different".

Reply:
Then could you instead put a message that this is dynamically set to at least 2x the size of the payload?

Reply:
Agree, added.